core/arch/instr.h

```diff
@@ -318,6 +318,13 @@ typedef struct _dr_instr_label_data_t {
     ptr_uint_t data[4]; /**< Generic fields for storing user-controlled data */
 } dr_instr_label_data_t;
 
+/**
+ * Label instruction callback function. Set by instr_set_label_callback() and
+ * called when the label is freed. \p instr is typically an instruction related
+ * to the label which can be freed at the same time as the label.
+ */
+typedef void (*instr_label_callback)(void *drcontext, instr_t *instr);
```

Comment:
instr_label_callback_t  for types?

Reply:
Oops! For some reason I didn't think this convention applied to callback functions but looking again it does.